ext/RMagick/rmpixel.c

```diff
@@ -136,8 +136,16 @@ Pixel_alpha(VALUE self)
  *
  * @param self this object
  * @return the opacity value
+ * @deprecated This method has been deprecated. Please use Pixel_alpha.
  */
-DEF_ATTR_READER(Pixel, opacity, int)
+VALUE
+Pixel_opacity(VALUE self)
+{
+    Pixel *pixel;
+    rb_warning("Pixel#opacity is deprecated; use Pixel#opacity.");
```

Comment:
@dlemstra looks like a typo here. Should this be `"use Pixel#alpha."`?

Reply:
Woops... thanks for the review and will submit a new PR to fix this.